spec/javascript/fixtures/short_form_submit_payload.js

```diff
@@ -25,6 +28,8 @@ const payload = {
        lastName: 'jones',
        middleName: null,
        address: '123 MAIN ST',
+       alternatePhone: null,
```

Comment:
can we update fixture to have these values non-null so we can trace them through and make sure they show up where expected in snapshot, etc?